content/applications/inventory_and_mrp/barcode/setup/serial_numbers_lots.rst

```diff
@@ -0,0 +1,138 @@
+===================================
+Barcodes for lot and serial numbers
+===================================
+
+The **Barcode** app saves time and prevents user errors while handling :doc:`lot
+<../../inventory/product_management/product_tracking/lots>` and :doc:`serial numbers
+<../../inventory/product_management/product_tracking/serial_numbers>`. Instead of typing long
+sequences of characters, print and scan barcodes instead.
+
+Configuration
+=============
+
+To use lot or serial numbers:
+
+#. They must be enabled in the database.
+#. Tracking by lot or serial number must be selected for each product.
+
+See the :doc:`serial numbers documentation
+<../../inventory/product_management/product_tracking/serial_numbers>` and :doc:`lots documentation
+<../../inventory/product_management/product_tracking/lots>` to learn how to start using this
+feature.
+
+GS1-compatible QR codes
+-----------------------
+
+:doc:`GS1 <../operations/gs1_nomenclature>` nomenclature can be used for serial numbers and lots.
+
+To enable GS1 barcodes for lots and serial numbers, go to :menuselection:`Inventory app -->
+Configuration --> Settings` and in the :guilabel:`Traceability` section under :guilabel:`Lots &
+Serial Numbers`, tick the box for :guilabel:`Print GS1 Barcodes for Lots & Serial Numbers`.
+
+.. image:: serial_numbers_lots/enable-GS1-barcodes.png
```

Comment:
Please make sure [media file names are written in lowercase](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files) :)
```suggestion
.. image:: serial_numbers_lots/enable-gs1-barcodes.png
```